tests/core/editable/manual/enterbr.html

```diff
@@ -0,0 +1,16 @@
+<h3>Copy below content:</h3>
+<p>a</p>
+<p>b</p>
+<br>
+<p id="editor"></p>
+
+<script>
+	if ( bender.tools.env.mobile ) {
+		bender.ignore();
```

Comment:
Good usage of `bender` tools. However, please notice, that the editor is created anyway. To achieve a little bit advanced ignore, take a look at https://github.com/ckeditor/ckeditor4/blob/major/tests/core/ckeditor/manual/hcdetection.html

The execution is stopped with a return statement: https://github.com/ckeditor/ckeditor4/blob/569ce5bae6f2305ee9aeffa7f961479c02da0420/tests/core/ckeditor/manual/hcdetection.html#L25

But it also requires to wrap the entire script in IIFE. Without this, the entire test will 'hang'.